src/PrimitiveObsession.Test/EngineContainerBuilderTest.cs

```diff
@@ -1,69 +0,0 @@
-﻿using System;

```

Comment:
Ora, secondo me, se non c'è business logic a distrarre il lettore, così che si possa concentrare sul tema di cui parliamo (come registrare delle istanze che hanno particolari tipi di dipendenze), insomma, se quello che fanno le classi internamente non è rilevante ai fini dell'esposizione, allora vale la pena di usare delle [Metasyntactic variables](https://en.wikipedia.org/wiki/Metasyntactic_variable) come `Foo`, `Bar`, `Baz` and `qux`.

Quando un programmatore in un articolo trova `Foo` e `Bar`, capisce che si sta facendo un esempio, e che l'esempio è generico, vale per qualsiasi classe, e sa che può tranquillamente ignorare l'implementazione interna.

Sicché, ho cambiato i nomi delle classi e le ho rese generiche.

Come nome della dipendenza primitiva ho usato qualcosa che i programmatori sanno essere normalmente implementato come primitivo, ed ho messo `ConnectionString`. Ho l'impressione che il nome precedente `TireCounter` o `TireBuilder` non aiutassero il lettore a capire al primo sguardo.

Ho usato una stringa, così dopo introduciamo il caso di una URL, anch'essa normalmente rappresentata con uno `string`.